packages/back-end/types/organization.d.ts

```diff
@@ -284,12 +283,4 @@ export type NamespaceUsage = Record<
   }[]
 >;
 
-export type ReqContext = {
-  org: OrganizationInterface;
-  userId: string;
-  email: string;
-  environments: string[];
-  userName: string;
-  readAccessFilter: ReadAccessFilter;
-  auditUser: EventAuditUser;
-};
+export type ReqContext = ReqContextInterface;
```

Comment:
Is there being done in order to prevent having to update ~100 files, or was there another determining factor here?

Reply:
Yes, exactly